src/main/java/teammates/ui/webapi/GetFeedbackSessionLogsAction.java

```diff
@@ -126,8 +126,20 @@ public JsonResult execute() {
                 throw new EntityNotFoundException("Feedback session not found");
```

Comment:
should we use the feedbackSessionId and studentId instead to check for if they exist?

Reply:
good catch! I think we should switch it to use ids instead for the rest of the method as well

Reply:
yup, we should use the id as far as possible if the course is migrated, I see that that is done already, nice work!